optimize-distro/assembly-distro-production.xml

```diff
@@ -20,7 +20,6 @@
       <directory>src/config/</directory>
```

Comment:
in the `config` folder you also need to remove the `OptimizeLicense.txt`, since this type of license is only applicable to C7